free-programming-books.md

```diff
@@ -1037,9 +1037,9 @@
 * [Elixir School](https://elixirschool.com) (HTML)
 * [Études for Elixir](http://chimera.labs.oreilly.com/books/1234000001642/index.html) (HTML)
 * [Getting Started Guide](http://elixir-lang.org/getting-started/introduction.html) (HTML) [(PDF, MOBI, EPUB)](https://github.com/potatogopher/elixir-getting-started)
+* [Joy of Elixir](https://joyofelixir.com) - (HTML)
```

Comment:
Would adding an author name be appropriate? see #2383 . Also, #2357 suggests a way to mark "in-progress" books.